src/Event/EventRegistrationCommandHandler.php

```diff
@@ -40,16 +36,10 @@ public function handle(EventRegistrationCommand $command): void
 
         $this->manager->create($registration = $this->factory->createFromCommand($command));
 
-        $this->mailer->sendMessage(EventRegistrationConfirmationMessage::createFromRegistration(
+        $this->dispatcher->dispatch(Events::EVENT_REGISTRATION_CREATED, new EventRegistrationCreatedEvent(
```

Comment:
Why do we need a dispatcher now ?

Reply:
Ok I saw the previous message of @HeahDude.

Reply:
IMO it's useless to send mail through a subscriber in this case. They do not bring anything here but complexity.

Reply:
SRP :)

Reply:
Maintainability and readability are giving you much more than SRP in that case.